tests/e2e/idps_test.go

```diff
@@ -763,6 +761,193 @@ var _ = Describe("TF Test", func() {
 			})
 		})
 
+		Describe("IDP reconciliation scenrios", func() {
+			var (
+				gitLabIdpName         string
+				gitlabIDPClientId     string
+				gitlabIDPClientSecret string
+				gitlabIdpOcmAPI       *cmv1.IdentityProviderBuilder
+			)
+
+			BeforeEach(func() {
+				idpService.ldap = *exe.NewIDPService(con.LdapDir)     // init new ldap service
+				idpService.gitlab = *exe.NewIDPService(con.GitlabDir) // init new gitlab service
+				gitlabIdpOcmAPI = cmv1.NewIdentityProvider()
+				gitlabIDPClientId = h.GenerateRandomStringWithSymbols(20)
+				gitlabIDPClientSecret = h.GenerateRandomStringWithSymbols(30)
+				userName = "newton"
+				password = "password"
+			})
+
+			Context("Author:smiron-Medium-OCP-65808 @OCP-65808 @smiron", func() {
+				It("OCP-65808 - IDP reconciliation basic flow",
+					ci.Day2, ci.Medium, ci.FeatureIDP, ci.Exclude, func() {
```

Comment:
is CI.Exclude label on purpose?

Reply:
yes, because of the` oc.Login` which causes failures and we yet to figure out how to resolve it